d2bs/kolbot/libs/common/Pather.js

```diff
@@ -895,6 +895,14 @@ ModeLoop:
 			}
 
 			if (me.inTown) {
+				if (me.area === 40 && me.x === 5153 && me.y === 5203) {
```

Comment:
instead of checking exact location, perhaps check distance from warriv < threshold -> use warriv